hack/ci/e2e.sh

```diff
@@ -187,9 +187,32 @@ EOF
         | grep -cv "node-role.kubernetes.io/master" \
     )"
 
-    # ginkgo regexes
-    SKIP="${SKIP:-}"
-    FOCUS="${FOCUS:-"\\[Conformance\\]"}"
+    # ginkgo regexes, use CI k/k presubmit regex by default
+    SKIP="${SKIP:-"\\[Slow\\]|\\[Disruptive\\]|\\[Flaky\\]|\\[Feature:.+\\]"}"
+    FOCUS="${FOCUS:-""}"
+
+    # skip known issues
+    # [sig-node] RuntimeClass should run a Pod requesting a RuntimeClass with a configured handler [NodeFeature:RuntimeHandler]
+    # https://github.com/kubernetes/kubernetes/issues/80098
+    SKIP="RuntimeClass.should.run.a.Pod.requesting.a.RuntimeClass.with.a.configured.handler|${SKIP}"
+    # [sig-storage] In-tree Volumes [Driver: nfs]
+    # [sig-storage] PersistentVolumes NFS
+    # TODO: kind nodes can't mount nfs volumes: install apt install -y nfs-common in the base image, kernel version, ...??
+    SKIP="In-tree.Volumes.\\[Driver:.nfs\\]|PersistentVolumes.NFS|${SKIP}"
+    # [sig-storage] PersistentVolumes-local
+    # [sig-storage] In-tree Volumes [Driver: local]
+    # fails because sh: 1: sudo: not found 
+    # Fix: https://github.com/kubernetes/kubernetes/pull/80135
+    SKIP="PersistentVolumes-local|In-tree.Volumes.\\[Driver:.local\\]|${SKIP}"
+    # [sig-auth] PodSecurityPolicy
```

Comment:
(noting that this is because we don't have PSP working properly in kind)

Reply:
I remember some conversations about that in the slack channel, IIRC you can´t have swap enable.
Is this something we can have working in current CI?

Reply:
CI doesn't have swap enabled because it is also kubernetes. swap relates to memory limits though, I think this is a container "nesting" issue.

Reply:
we didn´t enable the admission controller, in order to do that we need
```
diff --git a/pkg/cluster/internal/kubeadm/config.go b/pkg/cluster/internal/kubeadm/config.go
index eb87dfb..15e0f2f 100644
--- a/pkg/cluster/internal/kubeadm/config.go
+++ b/pkg/cluster/internal/kubeadm/config.go
@@ -99,6 +99,9 @@ controlPlaneEndpoint: "{{ .ControlPlaneEndpoint }}"
 api:
   advertiseAddress: "{{ .NodeAddress }}"
   bindPort: {{.APIBindPort}}
+# enable PodSecurityPolicy admission controller
+apiServerExtraArgs:
+  enable-admission-plugins: PodSecurityPolicy
 # we need nsswitch.conf so we use /etc/hosts
 # https://github.com/kubernetes/kubernetes/issues/69195
 apiServerExtraVolumes:
@@ -165,6 +168,9 @@ controlPlaneEndpoint: "{{ .ControlPlaneEndpoint }}"
 networking:
   podSubnet: "{{ .PodSubnet }}"
   serviceSubnet: "{{ .ServiceSubnet }}"
+# enable PodSecurityPolicy admission controller
+apiServerExtraArgs:
+  enable-admission-plugins: PodSecurityPolicy
 # we need nsswitch.conf so we use /etc/hosts
 # https://github.com/kubernetes/kubernetes/issues/69195
 apiServerExtraVolumes:
@@ -259,6 +265,8 @@ controlPlaneEndpoint: "{{ .ControlPlaneEndpoint }}"
 # to the cluster after rewriting the kubeconfig to point to localhost
 apiServer:
   certSANs: [localhost, "{{.APIServerAddress}}"]
+  extraArgs:
+    enable-admission-plugins:  PodSecurityPolicy,LimitRanger,ResourceQuota,AlwaysPullImages,DefaultStorageClass
 controllerManager:
   extraArgs:
     enable-hostpath-provisioner: "true"
@@ -356,6 +364,8 @@ controlPlaneEndpoint: "{{ .ControlPlaneEndpoint }}"
 # to the cluster after rewriting the kubeconfig to point to localhost
 apiServer:
   certSANs: [localhost, "{{.APIServerAddress}}"]
+  extraArgs:
+    enable-admission-plugins:  PodSecurityPolicy,LimitRanger,ResourceQuota,AlwaysPullImages,DefaultStorageClass
 controllerManager:
   extraArgs:
     enable-hostpath-provisioner: "true"
```
Problem is that host without apparmor don´t allow to create pods with apparmor annotations, and this breaks `kindnet`

Reply:
perhaps we should adjust the kindnetd spec

Reply:
but if we enable it, ir can break a lot of users that use manifests with the apparmor and seccomp annotations

Reply:
that's orthogonal to adjusting the kindnetd spec so that it _Could_ be enabled

Reply:
@BenTheElder check the last commit 1d0c656f1a9a8d791421d1446c52ab7980aff0f8

Reply:
Let me check tomorrow if it works locally and will submit a new PR with the change